### PR TITLE
[release-0.14] docs: mark v0.14 EOL'd

### PR DESCRIPTION
**Pull request by marquiz**

### CODE REVIEW

File: docs/_includes/templates/content.liquid [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        <button id="toggle" aria-label="Toggle menu" class="btn-octicon p-2 m-0 text-white" type="button">
+            <i class="fa fa-bars"></i>
+        </button>
+        <div class="title flex-1 d-flex flex-justify-center">
+            <a class="h4 no-underline py-1 px-2 rounded-1" href="{{ site.baseurl }}/">{{ site.title }}</a>
+        </div>
+    </div>
+    <div class="bg-red-2">
+        <div class="content p-3 px-sm-5">
+            This documentation is for Node Feature Discovery version that is no longer supported. Please upgrade and visit the
```
Comment (ArangoGutierrez) [inline docs/_includes/templates/content.liquid:12]:
```suggestion
            This documentation is for a Node Feature Discovery version that is no longer supported. Please upgrade and visit the
```

**Review by ArangoGutierrez — CHANGES_REQUESTED** [review]
just a small nit

File: docs/_includes/templates/content.liquid (hunk 1 above)
Comment (marquiz) [reply docs/_includes/templates/content.liquid]: Fixed as suggested

**Pull request merged**